tests/integration/conftest.py

```diff
@@ -19,7 +19,10 @@
 
 @pytest.fixture(scope="session")
 def test_repo_dir(tmp_path_factory: pytest.FixtureRequest) -> Path:
-    test_repo_url = "https://github.com/hermetoproject/integration-tests.git"
+    test_repo_url = os.environ.get(
+        "HERMETO_TEST_INTEGRATION_TESTS_REPO",
+        "https://github.com/hermetoproject/integration-tests.git",
```

Comment:
Ideally, this could be tweaked also on a per-test basis (I have that somewhere deep on my TODO list) otherwise you'd have to have all branches synced on your fork (I guess that's also fair to expect, but I don't have that). That's a change for another day though!

**Commit message:**
s|tests: use environment variable for test repository URL|tests: integration: Let test repo URL be configurable with an env variable

Reply:
Non-actionable, unrelated to this PR: repository used to be a part of test parameters some time ago, then it got refactored to a global value. Maybe it is time to refactor it back in?

Reply:
> Non-actionable, unrelated to this PR: repository used to be a part of test parameters some time ago, then it got refactored to a global value. Maybe it is time to refactor it back in?

It still should be a global default value unless overridden which wasn't the case before I think, anyway, we should create an issue for that.

Reply:
mmm ok, I can just drop this commit then

Reply:
If it weren't clear from my message, this is still an improvement, just the commit message needs to be adjusted a little.